pkg/vdr/web/resolver.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package web
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdr"
+)
+
+// Read resolves a did:web did.
+func (v *VDR) Read(didID string, opts ...vdr.ResolveOpts) (*did.Doc, error) {
+	// apply resolve opts
+	docOpts := &vdr.ResolveDIDOpts{
+		HTTPClient: &http.Client{},
+	}
+
+	for _, opt := range opts {
+		opt(docOpts)
+	}
+
+	url, err := parseDIDWeb(didID)
+	if err != nil {
+		return nil, fmt.Errorf("error resolving did:web did --> could not parse did:web did --> %w", err)
+	}
+
+	resp, err := docOpts.HTTPClient.Get(url)
+	if err != nil {
+		return nil, fmt.Errorf("error resolving did:web did --> http request unsuccessful --> %w", err)
+	}
+
+	defer closeResponseBody(resp.Body)
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, fmt.Errorf("error resolving did:web did --> error reading http response body: %s --> %w", body, err)
+	}
+
+	doc, err := did.ParseDocument(body)
+	if err != nil {
+		return nil, fmt.Errorf("error resolving did:web did --> error parsing did doc --> %w", err)
+	}
+
+	return doc, nil
+}
+
+func closeResponseBody(respBody io.Closer) {
+	logger := log.New("aries-framework/pkg/vdr/web")
```

Comment:
I would still declare logger globally so next time you need it - it will be ready to use.

Reply:
ah okay, I hadn't seen this line in the lint file

  exclude:
    # Allow package logger variables (for now)
    - \`logger\` is a global variable